spec/board_spec.rb

```diff
@@ -8,20 +8,30 @@
     expect(player_board.board).to eq([%w[- - -], %w[- - -], %w[- - -]])
   end
 
-  it 'expects number of blank spaces to be 9' do
-    expect(player_board.num_of_blank_spaces).to eq 9
-  end
-
-  it 'creates a hash from array of arrays' do
-    expect(player_board.make_free_positions_hash([[1, 2]])).to eq({ 1 => { row: 1, col: 2 } })
-  end
-  it 'prints the board' do
-    player_board.print_board(output)
-    expect(output.string).to include("[\"-\", \"-\", \"-\"]\n")
-  end
-
   it "adds the player's symbol to the board's specified index " do
     player_board.apply_move('X', { row: 0, col: 0 })
     expect(player_board.board).to eq([%w[X - -], %w[- - -], %w[- - -]])
   end
+  describe '#board_state' do
+    context 'when the game is won' do
```

Comment:
I would also include cases that check when the game is not over yet or tied for completeness.

Reply:
Resolved at 5fbd8db67c30a860f9e237172e7fd1152cb36f01